SecurityPkg/Include/Library/OemTpm2InitLib.h

```diff
@@ -0,0 +1,127 @@
+/** @file -- OemTpm2InitLib.h
+Oem library hooks for additional TPM 2.0 initialization.
+
+MS_CHANGE_?
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _OEM_TPM_2_INIT_LIB_H_
+#define _OEM_TPM_2_INIT_LIB_H_
+
+// In order to include this header, you will also need to include.
+// - Pi/PiBootMode.h
+// - Protocol/Tcg2Protocol.h

```

Comment:
Why not just include those headers in this header?

Reply:
its a good question. Will change that.